cli/crates/cli/src/create.rs

```diff
@@ -25,15 +25,38 @@ impl Display for RegionSelection {
     }
 }
 
+#[allow(clippy::module_name_repetitions)]
+pub struct CreateArguments<'a> {
+    pub account_slug: &'a str,
+    pub name: &'a str,
+    pub regions: Vec<&'a str>,
+}
+
 /// # Errors
 #[tokio::main]
-pub async fn create() -> Result<(), CliError> {
+pub async fn create(arguments: &Option<CreateArguments<'_>>) -> Result<(), CliError> {
     let environment = Environment::get();
 
     let (accounts, available_regions, closest_region) = create::get_viewer_data_for_creation()
         .await
         .map_err(CliError::BackendApiError)?;
 
+    if let Some(arguments) = arguments {
+        let account_id = accounts
+            .into_iter()
+            .find(|account| account.slug == arguments.account_slug)
+            .ok_or(CliError::NoAccountFound)?
+            .id;
+
+        let domains = create::create(&account_id, arguments.name, &arguments.regions)
+            .await
+            .map_err(CliError::BackendApiError)?;
+
+        report::created(arguments.name, &domains);
+
+        return Ok(());
+    }
+
     let options: Vec<AccountSelection> = accounts.into_iter().map(AccountSelection).collect();
```

Comment:
Should we split this to another function? Then we can do `if let ... else { interactive_flow }`...